code-challenges.js

```diff
@@ -1,54 +1,143 @@
 
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
+  // This function needs a single param/argument and to output one of three predetermined messages based on three ways to compare the input with the number 212
+  // A single if statement with two chaining else ifs covers all these cases
+
+  const boilingChecker = (temp) => {
+    if(temp > 212){
+      console.log(`${temp} is above boiling point`)
+    } else if(temp < 212){
+      console.log(`${temp} is below boiling point`)
+    } else if(temp == 212){
+      console.log(`${temp} is at boiling point`)
+    }
+  }
+
+ const boilingPoint = (temp) => {
+   return (temp == 212)
+     ? '212 is at boiling point'
+     : (temp > 212)
+     ? `${temp} is above boiling point`
+     : `${temp} is below boiling point`
+ }
+console.log(boilingPoint(temp1))
+console.log(boilingPoint(temp2))
+console.log(boilingPoint(temp3))
 
 // --------------------2) Create a function that takes in two arrays of numbers and returns a single array with all the values sorted least to greatest.
 // Use the test variables provided below. Expected output: [-9, -7, 0, 3, 7, 8, 9, 13, 22, 36]
 
 var myNumbers1 = [3, 7, 0, 36, -9]
 var myNumbers2 = [8, -7, 22, 9, 13]
 
-
+  // This function has two parameters and returns one array. Both arguments are expected to be arrays. Simply return the output of the arrays operated on first by the
+  // array.concat() method, and then the array.sort() method
+  /* const arrayJoiner = (array1, array2) => {
+  //   console.log(array1.concat(array2).sort())
+   }*/
+  // output
+  // [
+  //   -7, -9, 0, 13, 22,
+  //    3, 36, 7,  8,  9
+  // ]
+  //
+  // After reading more about array.sort() I understand how to make it do what I need it to do and sort numerically
+
+  const arrayJoiner = (array1, array2) => {
+    console.log(array1.concat(array2).sort((a, b)=>a-b))
+  }
+
+  // arrayJoiner(myNumbers1, myNumbers2)
+  // output
+  // [
+  //   -9, -7,  0,  3,  7,
+  //    8,  9, 13, 22, 36
+  // ]
+  //
+  // But I don't think that's what you necessarily wanted. Nevertheless, I do not know any better way to sort the array numerically, especially without looping.
 
 // --------------------3) Create a function that takes in a string of a single word and returns the string with all letters reversed.
 // Use the test variables provided below. Expected output: "ovarb", "eilrahc"
 
 var myString1 = "bravo"
 var myString2 = "charlie"
 
+  // If I treat the string like an array this problem is quite simple, as I can just use the .reverse() method on an array and then stringify it once again, thus
+  // I'll use the .split() function on the string input and go from there
 
+  const reverseString = (string) => {
+    let array = string.split("")
+    array.reverse()
+    array = array.join("")
+    return array
+  }
+
+  // console.log(reverseString(myString1))
+  // It works! I had to sit and remember for a few seconds that the str.split() function needs an empty pair of quotation marks
```

Comment:
Good work! The way you can augment the data with the split and join can get a bit intense.